lang-ca.js

```diff
@@ -1187,29 +1219,31 @@ SnapTranslator.dict.ca = {
 
     // math functions
     'abs':
-        'abs',
+        'valor absolut',
     'floor':
         'part entera',
     'sqrt':
         'arrel quadrada',
     'sin':
-        'sin',
+        'sinus',
```

Comment:
You're right, I got carried away because I noticed I had left "abs" in there (and we don't usually use that abbreviation), so I changed that one and kept going. I'm correcting the rest.

Thanks for noticing! :)